client/build.gradle

```diff
@@ -52,6 +52,16 @@ idea {
     }
 }
 
+jar {
```

Comment:
WDYT about adding it to `subprojects` of the main build file? So that every JAR gets it, not just client/testing

Reply:
will that affect plugins? or should I exclude them explicitly?

Reply:
moved